docs/installation.rst

```diff
@@ -71,10 +77,36 @@ whats happening. The GladierClient comes with some built-in functions to help wi
 a nice way to watch progression as the flow executes. Once the flow has finished,
 ``get_details()`` will fetch output from the Globus Flow, so it can be displayed in a
```

Comment:
GladierClient.get_details() not sure if this is needed but it will be nice to add that, since you can have many flows running on the same client. some verbose may be needed

Reply:
I'll add that